test/export/test_export.py

```diff
@@ -6075,7 +6075,7 @@ def forward(self, x):
             """\
 def forward(self, x):
     cos = torch.ops.aten.cos.default(x)
-    auto_functionalized = torch.ops.higher_order.auto_functionalized(torch.ops.testlib.foo.default, x = x, z = cos);  x = cos = None
+    auto_functionalized = torch.ops.higher_order.auto_functionalized(torch.ops.testlib.foo.default, _x_base_index = 0, _z_base_index = 1, _all_bases = [x, cos]);  x = cos = None
```

Comment:
@angelayi @SherlockNoMad we are changing how the auto_functionalized HOP works. Are there any internal BC/FC concerns? (does anyone internally actually have a serialized exported program with auto_functionalized in it?)

Reply:
answer might be yes, I see use cases in sigmoid and executorch

Reply:
btw can this diff have (not user facing label?) if not whats the landing proccess never had a diff with out that

Reply:
Yes, we have prod model using auto_functionalization HOP. 
This will break BC.

Reply:
To evolve, you will need to implement it as auto_functionalized_v2

Reply:
@laithsakka regarding labels, anything that begins with "release notes: " is sufficient to land. For this one we probably want release notes: inductor